README.md

```diff
@@ -0,0 +1,38 @@
+## node reference
+
+Outline transport agnostic node js service. 
+
+
+directory layout 
+
+```sh
+.
+├── config  
+├── context # generated contexts generated each time
+├── deploy # our business logic not generated 
+└── endpoints
+    └── http #generated endpoints. Each endpoint only generated once    
+
+```
+
+
+files layout 
+
+```sh 
+├── config
+│   ├── configValidation.js
+│   └── dev.json
+├── context
+│   └── deploy.js
+├── deploy # directory named after the domain the business logic covers 
```

Comment:
all the other directory names are very generic, but this one is very specific to the business domain of the app. If there are many services, the top level dir could become a little bit messy. Maybe create a `service` dir and put the domain business logic there?